Compatibility/MainWindow.xaml.cs

```diff
@@ -0,0 +1,301 @@
+        }
+
+        private bool TryGetCompatibilityPreferenceSwitchValue(string compatPreferenceClass, string switchName, out bool switchValue)
+        {
+            switchValue = false;
+            Type tSwitch = GetCompatPreferenceType(compatPreferenceClass);
+            if (tSwitch == null)
+            {
+                return false;
+            }
+
+            var pProperty = tSwitch.GetProperty(switchName, BindingFlags.Static | BindingFlags.Public);
+            if (pProperty == null)
+            {
+                pProperty = tSwitch.GetProperty(switchName, BindingFlags.Static | BindingFlags.NonPublic);
+            }
+
+            if (pProperty != null)
+            {
+                try
+                {
+                    switchValue = (bool)pProperty.GetValue(null);
+                    return true;
+                }
+                catch
+                {
+                }
+            }
+
+            return false;
+        }
+
+        private bool TryGetCompatibilityPreferenceSwitchValue(string compatPreferenceClass, string switchName, out string switchString)
+        {
+            switchString = null;
+            Type tSwitch = GetCompatPreferenceType(compatPreferenceClass);
+            if (tSwitch == null)
+            {
+                return false;
+            }
+
+            var pProperty = tSwitch.GetProperty(switchName, BindingFlags.Static | BindingFlags.Public);
+            if (pProperty == null)
+            {
+                pProperty = tSwitch.GetProperty(switchName, BindingFlags.Static | BindingFlags.NonPublic);
+            }
+
+            if (pProperty != null)
+            {
+                try
+                {
+                    object o = pProperty.GetValue(null);
+                    if (pProperty.PropertyType == typeof(bool?))
+                    {
+                        bool? b = (bool?)o;
+                        if (!b.HasValue)
+                        {
+                            switchString = "(bool?)null";
+                        }
+                        else
+                        {
+                            switchString = $"(bool?){b.Value.ToString()}";
+                        }
+                        return true;
+                    }
+
+                    switchString = o.ToString();
+                    return true;
+                }
+                catch
+                {
+                }
+            }
+
+            return false;
+        }
+
+        private void Window_Loaded(object sender, RoutedEventArgs e)
+        {
+            PopulateAppContextInformation();
+        }
+
+        private void PopulateAppContextInformation()
+        {
+            string[] appContextClasses =
+                {
+                "BaseAppContextSwitches",
+                "CoreAppContextSwitches",
+                "FrameworkAppContextSwitches",
+                "AccessibilitySwitches",
+                //"BuildTasksAppContextSwitches"
```

Comment:
Do you want to leave this here commented out?  I assume there is no great way of accessing the PBT location and loading the DLL at runtime so you can scan these.

Reply:
Yeah, leaving this commented out is intentional. PBT is not accessible in in netcore since we took it out of the runtime set.